packages/logos-docusaurus-theme/src/client/components/mdx/Grid/Grid.tsx

```diff
@@ -158,7 +207,27 @@ const GridRoot = styled.div<{
       'down',
     )(css`
       .mdx-grid__scroll {
-        display: none !important;
+        display: ${props.xs?.scrollButtons ? 'flex' : 'none'};
+
```

Comment:
`scrollButtons` property should work consistently with other breakpoint options. Please take a look at how other options are implemented: https://github.com/acid-info/logos-docusaurus-plugins/blob/main/packages/logos-docusaurus-theme/src/client/components/mdx/Grid/Grid.tsx#L113-L151